### ISSUE TITLE
Comanches vet up extremely quick (and give off little XP)

### ISSUE THREAD

**Issue by ExiLe-ZH** [TheSuperHackers/GeneralsGamePatch#1016]
Compared to other meta units, Commies vet up extremely quick because of the strong onboard weapons which are capable of leveling buildings very quickly (buildings give of alot of XP). Whereas Vees or Lixes often rely on their passengers to make the kills, which makes a passenger vet up , but not the vehicle itself.

If you further compare Commies to other killing machines like **Overlords** and **Alphas** we can see the requirement is very low, while the Commies in groups are much more efficient usually.

Same story for the ExperienceValue, vetted Comanches are incredibly hard to catch, unless the Air player commits to a bad engagement or doesn't pay attention to get away in time.

It seems EA wasn't really aware of how capable Comanches are.

```
Comanche:
  ExperienceValue     = 50 50 100 200   ; Experience point value at each level
  ExperienceRequired  = 0 100 200 400   ; Experience points needed to gain each level

Helix:
  ExperienceValue     = 50 100 150 200 ;Experience point value at each level
  ExperienceRequired  = 0 100 200 400  ;Experience points needed to gain each level

Tank Hunter/MD:
  ExperienceValue = 20 20 40 60    ;Experience point value at each level
  ExperienceRequired = 0 100 200 400  ;Experience points needed to gain each level

Vee:
  ExperienceValue = 50 50 100 150   ;Experience point value at each level
  ExperienceRequired = 0 100 150 300  ;Experience points needed to gain each level

Crusader/Paladin:
  ExperienceValue        = 100 100 200 400 ;Experience point value at each level
  ExperienceRequired     = 0 200 300 600 ;Experience points needed to gain each level

Crusader/Paladin Patched:
  ExperienceValue        = 100 100 200 300 ;Experience point value at each level
  ExperienceRequired     = 0 200 300 600 ;Experience points needed to gain each level

Overlord:
  ExperienceValue = 200 200 400 600   ;Experience point value at each level
  ExperienceRequired = 0 400 600 1200  ;Experience points needed to gain each level

Alpha Aurora:
  ExperienceValue = 200 200 400 400   ;Experience point value at each level
  ExperienceRequired = 0 200 400 800   ;Experience points needed to gain each level
```

### Proposal:
Since buffing is prioritized with the gamepatch approach, +25% is atleast a balance improvement over the original.

```
ExperienceValue     = 60 60 125 250 ;Experience point value at each level
ExperienceRequired = 0 125 250 500   ;Experience points needed to gain each level
```
Although +50% is more in line with other units:

```
ExperienceValue     = 75 75 150 300 ;Experience point value at each level
ExperienceRequired = 0 150 300 600   ;Experience points needed to gain each level
```

**Comment by commy2** [comment]
I don't think this is an issue with vComanches, which are rather bad in most circumstances.

If AFG Comanches are hard to kill, then that is because of their stupendous armor, especially after Countermeasures and the fact that they can cloak despite taking damage by just stopping to fire. Especially the armor stacks with veterancy health bonuses.

**Comment by ExiLe-ZH** [comment]
> I don't think this is an issue with vComanches, which are rather bad in most circumstances.
> 
> If AFG Comanches are hard to kill, then that is because of their stupendous armor, especially after Countermeasures and the fact that they can cloak despite taking damage by just stopping to fire. Especially the armor stacks with veterancy health bonuses.

True, since they're different they can be treated differently aswell I guess.

**Comment by Jundiyy** [comment]
Air ones also have different textures, they look different.

Btw, the bottom of your first message, how is that a buff? Looks like nerf to me.

I don't really want to nerf, but it was done, I'd rather just increase the exp required, not the exp value. 125 version seems okay.

**Comment by ExiLe-ZH** [comment]
It's a nerf, hence I'm proposing less of a nerf +25% instead of the +50% (the model that actually would make the most sense when comparing with other units).

**Comment by ReLaX82** [comment]
What about changing the stealth abilities a little?

**Comment by ExiLe-ZH** [comment]
> What about changing the stealth abilities a little?

In what way?

**Comment by ReLaX82** [comment]
Longer time to stealth again. I can imagine this has a big impacts when they get unstealthed by Radar van or whatever. But your other points are true, too.
